src/controllers/productsController.js

```diff
@@ -11,7 +11,22 @@ const getAllProducts = async (req, res) => {
   }
 };
 
+const getSpecificProducts = async (req, res) => {
```

Comment:
함수 이름은 이 정도만 되도 될 것 같습니다.
```suggestion
const getProduct = async (req, res) => {
```